File: 프로그래머스/1/12901. 2016년/2016년.py
def solution(a, b):
    days = ["FRI", "SAT", "SUN", "MON", "TUE", "WED", "THU"]
    
    month_days = [31, 29, 31, 30, 31, 30, 
                  31, 31, 30, 31, 30, 31]
    
    total = 0
    
    for i in range(a - 1):
        total += month_days[i]
    
    total += b - 1
    
    return days[total % 7]

File: 프로그래머스/1/12901. 2016년/2016년.cpp
#include <string>
#include <vector>

using namespace std;

string solution(int a, int b) {
    vector<int> days = {
        31, 29, 31, 30, 31, 30,
        31, 31, 30, 31, 30, 31
    };
    
    vector<string> week = {
        "FRI", "SAT", "SUN", "MON",
        "TUE", "WED", "THU"
    };
    
    int total = 0;
    
    for (int i = 0; i < a - 1; i++) {
        total += days[i];
    }
    
    total += b - 1;
    
    return week[total % 7];
}